src/OptionsPopup.qml

```diff
@@ -635,6 +637,18 @@ Window {
             }
         }
 
+        var hwFilterList = imageWriter.getHWFilterList()
+        var hwFilterIsModelZero = imageWriter.getHWFilterIsModelZero()
+
+        if (hwFilterList) {
+            var targetTags = ["pi5-64bit", "pi4-64bit", "pi5-32bit", "pi4-32bit"]
```

Comment:
@tdewey-rpi Would it make sense for this to be added as an additional property in the JSON file, rather than having it hardcoded inside Imager itself? (e.g. perhaps a future Raspberry Pi 7 might also support USB-OTG :wink: )

Reply:
Here an example of some devices using the new scheme:
```json
{

"devices": [
      {
        "name": "Raspberry Pi 5",
        "tags": [
          "pi5-64bit",
          "pi5-32bit"
        ],
        "icon": "https://downloads.raspberrypi.com/imager/icons/RPi_5.png",
        "description": "The latest Raspberry Pi, Raspberry Pi 5",
        "matching_type": "exclusive",
        "features": [
            "ether_gadget"
        ]
      },
      {
        "name": "Raspberry Pi 4",
        "tags": [
          "pi4-64bit",
          "pi4-32bit"
        ],
        "default": false,
        "icon": "https://downloads.raspberrypi.com/imager/icons/RPi_4.png",
        "description": "Models B, 400, and Compute Modules 4, 4S",
        "matching_type": "inclusive",
        "features": [
            "ether_gadget"
        ]
      }
]

}
```

Reply:
I like it. We can also then use those feature flags in an AND with feature flags exposed by the OS to identify if a feature should be supported.

Reply:
What about renaming this to `supportedFeatures` so it doesn't implicate that this OS or this device has this feature?

Reply:
`capabilities` ?

Reply:
Also, given that this is describing a feature of the hardware, perhaps it should be named `"usb_otg"` rather than `"ether_gadget"` ?

Reply:
Completely agree features and also supported features seems a bit confusing. 
And `usb_otg` would also have the advantage that it can also be used for other features in the future to detect if a device OS combi does have support for it rather than checking if it supports `ether_gadget`.